crates/proto/src/rr/rdata/caa.rs

```diff
@@ -134,15 +140,41 @@ impl CAA {
         self.tag = tag;
     }
 
-    /// a potentially associated value with the property tag, see struct documentation
+    /// A value associated with the property tag, see struct documentation
```

Comment:
Maybe `#[deprecate]` this, referring to the new methods you've added, and file an issue to follow up on this for the next semver-incompatible release?